manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: tushev.org.AJUpdateWatcher
+PackageName: AJUpdateWatcher
+Moniker: aojdk-updatewatcher, ajupdatewatcher
```

Only one moniker is allowed
```suggestion
Moniker: aojdk-updatewatcher
```